root/components/home/Explore.js

```diff
@@ -0,0 +1,172 @@
+                  meet for discussions about the project. 
+                  We invite you to join if you have any questions or comments!
+                `)}
+              </p>
+            </div>
+          </div>
+        </div>
+        <div className="col-md-6 d-flex">
+          <div className="card p-4 m-4 shadow">
+            <div className="card-body">
+              <h5 className="text-center mb-4 fs-3 fw-bold">
+                <a
+                  href="/doc/Development"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('Development')}
+                </a>
+              </h5>
+              <p className="card-text fs-4">
+                {l(`
+                  Our developer resources will help you make
+                  use of our data or contribute code. If you are a
+                  commercial user, our live data feed will keep
+                  your local database in sync.
+                `)}
+              </p>
+            </div>
+          </div>
+        </div>
+        <div className="col-md-6 d-flex">
+          <div className="card p-4 m-4 shadow">
+            <div className="card-body">
+              <h5 className="text-center mb-4 fs-3 fw-bold">
+                <a
+                  href="/doc/About/History"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('History')}
+                </a>
+              </h5>
+              <p className="card-text fs-4">
+                {l(`
+                  Robert Kaye founded MusicBrainz as a
+                  free open music database. The project has since grown
+                  rapidly from a one-man operation to an international
+                  community of enthusiasts that appreciate
+                  both music and music metadata.
+                `)}
+              </p>
+            </div>
+          </div>
+        </div>
+
+        <div className="col-md-6 d-flex">
+          <div className="card p-4 m-4 shadow">
+            <div className="card-body">
+              <h5 className="text-center mb-4 fs-3 fw-bold">
+                <a
+                  href="/doc/MusicBrainz_Database"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('The Database')}
+                </a>
+              </h5>
+              <p className="card-text fs-4">
+                {l(`
+                  The MusicBrainz Database stores all of the
+                  various pieces of information we collect
+                  about music, from artists and their
+                  releases to works and their composers,
+                  and much more.
+                  All our data is available for commercial licensing.
+                  If you are interested in licensing this data
+                  for commercial use, please contact us.
+                `)}
```

Comment:
I'd change "please contact us" to "please get in touch" and link to https://metabrainz.org/supporters/account-type (arguably that page URL should be supporters/sign-up instead, but that's a different issue). Also, given that page also applies for non-commercial use of the data feed and whatnot, I'd say, altogether:

If you are interested in using this data in your organization, please {meb_signup|get in touch}. Our data is available for commercial licensing.

(I removed "all" because we don't really license the Cover Art Archive, but YMMV on the best wording here)

Reply:
Makes sense.